### PR TITLE
add SWR fetch function, ArtPieces and ArtPiecePreview Components

### PR DESCRIPTION
**Pull request by stekug**
add basic styling

### CODE REVIEW

File: components/ArtPiecePreview/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+export default function ArtPiecePreview({ piece }) {
+  const { imageSource: image, artist: artist, name: title } = piece;
+  return (
+    <StyledContainer>
+      <Image
+        src={image}
+        width={360}
+        height={0}
+        style={{ width: "auto", height: "auto" }}
```
Comment (gunnarmon) [inline components/ArtPiecePreview/index.js:27]: i'll copy that one! (:

**Review by gunnarmon — COMMENTED** [review]
seems good!

File: components/ArtPiecePreview/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  font-size: 1rem;
+  color: hotpink;
+`;
+
+const StyledTitle = styled.h2`
+  font-size: 1.2rem;
+`;
+
+export default function ArtPiecePreview({ piece }) {
+  const { imageSource: image, artist: artist, name: title } = piece;
```
Comment (stepris) [inline components/ArtPiecePreview/index.js:20]: Nicely done!

File: components/ArtPieces/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import ArtPiecePreview from "../ArtPiecePreview";
+
+export default function ArtPieces({ pieces }) {
+  console.log(pieces);
+
+  return (
+    <ul>
+      {pieces.map((piece) => (
+        <ArtPiecePreview key={piece.slug} piece={piece} />
```
Comment (stepris) [inline components/ArtPieces/index.js:9]: Interesting approach, directly using another component for the list items ✅

**Review by stepris — APPROVED** [review]
Everything's fine and dandy, ready for merging!

**Pull request merged**